app-emulation/virtualbox-guest-additions/virtualbox-guest-additions-7.0.2.ebuild

```diff
@@ -6,55 +6,60 @@ EAPI=8
 inherit edo linux-mod systemd toolchain-funcs udev
 
 MY_PN="VirtualBox"
-MY_PV="${PV/beta/BETA}"
-MY_PV="${MY_PV/rc/RC}"
-MY_P="${MY_PN}-${MY_PV}"
-[[ "${PV}" == *a ]] && DIR_PV="$(ver_cut 1-3)"
+MY_P="${MY_PN}-${PV}"
 
 DESCRIPTION="VirtualBox kernel modules and user-space tools for Gentoo guests"
 HOMEPAGE="https://www.virtualbox.org/"
-SRC_URI="https://download.virtualbox.org/virtualbox/${DIR_PV:-${MY_PV}}/${MY_P}.tar.bz2
+SRC_URI="https://download.virtualbox.org/virtualbox/${PV}/${MY_P}.tar.bz2
 	https://gitweb.gentoo.org/proj/virtualbox-patches.git/snapshot/virtualbox-patches-6.1.36.tar.bz2"
 
-LICENSE="GPL-2"
+# Reminder: see the LICENSE related comment in app-emulation/virtualbox-additions ebuild
+LICENSE="GPL-2+ GPL-3 LGPL-2.1 MIT"
 SLOT="0/$(ver_cut 1-2)"
-[[ "${PV}" == *_beta* ]] || [[ "${PV}" == *_rc* ]] || \
-KEYWORDS="amd64 x86"
-IUSE="X"
+KEYWORDS="~amd64 ~x86"
+IUSE="+dbus X"
 
 # automount Error: VBoxServiceAutoMountWorker: Group "vboxsf" does not exist
+# TODO: find out what this is, remove comment if obsolete
+
 RDEPEND="
 	acct-group/vboxguest
 	acct-group/vboxsf
 	acct-user/vboxguest
-	X? ( x11-apps/xrandr
+	sys-libs/zlib
+	dbus? ( sys-apps/dbus )
+	X? (
+		x11-apps/xrandr
 		x11-apps/xrefresh
-		x11-libs/libXmu
 		x11-libs/libX11
-		x11-libs/libXt
 		x11-libs/libXext
-		x11-libs/libXau
-		x11-libs/libXdmcp
-		x11-libs/libSM
-		x11-libs/libICE )
-	sys-apps/dbus
+		x11-libs/libXmu
+		x11-libs/libXt
+	)
 "
 DEPEND="
 	${RDEPEND}
-	>=dev-util/kbuild-0.1.9998.3127
+	x11-libs/libICE
```

Comment:
sorry to keep asking sutff: you're really sure these are DEPEND only? It's unusual, although I've seen some X packages with stuff like this before. PAM is particularly odd... is it because some file doesn't get installed?